src/libraries/System.Security.Cryptography/tests/X509Certificates/ExportTests.cs

```diff
@@ -67,7 +67,7 @@ public static void ExportAsPfx()
                 byte[] pfx = c1.Export(X509ContentType.Pkcs12);
                 Assert.Equal(X509ContentType.Pkcs12, X509Certificate2.GetCertContentType(pfx));
 
-                using (X509Certificate2 c2 = new X509Certificate2(pfx))
+                using (X509Certificate2 c2 = new X509Certificate2(pfx, (string?)null))
```

Comment:
I'd probably have gone with `""` or `string.Empty` just to avoid having to cast null.

```suggestion
                using (X509Certificate2 c2 = new X509Certificate2(pfx, ""))
```